src/lib/dateRangeHelpers.test.ts

```diff
@@ -0,0 +1,24 @@
+import { DateTime } from 'luxon';
+import { getTwelveMonthReportDateRange } from './dateRangeHelpers';
+
+describe('getTwelveMonthReportDateRange', () => {
+  it('should return startDate and endDate as ISO strings', () => {
+    const { startDate, endDate } = getTwelveMonthReportDateRange();
```

Comment:
Can you add tests that check the actual start and end date values? You can do `Settings.now = () => new Date(2024, 7, 31).valueOf();` to mock the current date. I'd like to see at least three tests: one for the first day of the month (e.g. May 1), a middle day of the month (e.g. May 15), and the last day of the month (e.g. May 31). That will help ensure that our logic works for any day.